app/models/views/office_business_entity_state.rb

```diff
@@ -0,0 +1,54 @@
+module Views
+  class OfficeBusinessEntityState
```

Comment:
@novotnyjakub do you mean that a specific feature is missing from [here](https://github.com/ministryofjustice/fr-staffapp/pull/554/files#diff-0de8e4919a5e91498a48885f86d0454bR4)